build/setup-kubectl.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
```

Comment:
Prior to the PR, when I attempted to build an image for restorationjob it would fail when executing the last RUN command that runs the 2 sh files.  I concluded that bash was not installed in the base image so just changed the two script's to use sh instead.  No idea what "ash" is either?

You are welcome to change it back to bash, or remove my change - I'm really busy at the moment.  I would expect to be able to build the image locally though, yes/no?

Is bash supposed to be installed?  Just not sure how the image ever built on the commit I pulled is all.

Reply:
💡cool, “ash” is default shell for alpine (ash is minimalist shell and sometimes keeps security by design) : https://wiki.alpinelinux.org/wiki/Change_default_shell

Since it is unintentional let’s revert it, I have few things to do but we can collaborate on this and I can remove this line sompoint today.

I can relive this in couple of hours, might add PR to your fork!!

Reply:
Cool, thanks for boosting my knowledge re "ash".  Seems like I would have been better changing both to "ash" is my guess.  Oh well never mind.

Thanks for understanding re availability.

Reply:
👍 cool, Fyi @markphillips100 - I have removed the other file discussed in one of comment and also reverted this change to whatever was there in main. :) please take a look and see if anything else got checked in which is not needed?

Reply:
Looks good @Tatsinnit .  As for `launchSettings.json`, that is not needed.  My bad.

Reply:
All good @markphillips100 - done, 👍 - please take a look and I've removed the other local launch setting file as well. Thanks.

Reply:
Nope.  no need for any of the launchSettings.json.  Visual Studio added them for me for some reason and I failed to notice.

Reply:
I confirm current changes building okay

Reply:
Awesome, done, I have removed and pushed this file as well `src/devhostagent/Properties/launchSettings.json` ! thanks will it be cool if you check that all looks sane at your end whenever you have time. Thanks so much for this.

Reply:
Yeah I just got out of meetings (thankfully) so was able to check a few minutes ago, and can confirm your latest builds and tests good also.

Reply:
Hi! Sorry this is still opened, I just wanted to give you an update. Our testing team found one scenario that did not work. In particular, restoration job was having issues. We are working on it before merging the change.

Reply:
Anything I can do?

Reply:
You are all good. I worked on this yesterday and fix the issue (it was on our end - docker images on dev environment had been overridden). Testing team gave green light for this change now.